### ISSUE TITLE
Support for PolyMC

### ISSUE THREAD

**Issue by thebitstick** [jrbudda/Vivecraft_118#37]
[PolyMC](https://polymc.org) is a fork of [MultiMC](https://multimc.org).

Detection should be nearly identical if not completely identical.

**Comment by I-Have-An-Issue** [comment]
+1

**Comment by nrgill28** [comment]
For those of you wanting to install Vivecraft into a PolyMC profile I have the workaround.

1. As per the [MultiMC install guide](https://www.vivecraft.org/multimc/) make sure you create your profile in PolyMC first and then **CLOSE POLYMC**.
2. Open the PolyMC data folder (`%AppData%\PolyMC\`) and here you need to do two things:
    - Create an empty file in the directory named `multimc.exe`
    - Rename `polymc.cfg` to `multimc.cfg`

**Comment by luximus-hunter** [comment]
> For those of you wanting to install Vivecraft into a PolyMC profile I have the workaround.

As awesome as that is, It'd ve amazing if it can detect PolyMC out of the box too

**Comment by nrgill28** [comment]
> As awesome as that is, It'd ve amazing if it can detect PolyMC out of the box too

Yes, of course having it be supported in some form would be infinitely better. But until then, at least the workaround isn't too crazy.
